.github/workflows/cicd.yaml

```diff
@@ -194,6 +195,7 @@ jobs:
         if: ${{ github.event_name == 'push' }}
         run: |
           tar czf ${{ env.OPENMLDB_PREFIX }}.tar.gz ${{ env.OPENMLDB_PREFIX }}/
+          sha256sum ${{ env.OPENMLDB_PREFIX }}.tar.gz > ${{ env.OPENMLDB_PREFIX }}-sha256sum
```

Comment:
pls ref https://github.com/4paradigm/hybridsql-asserts/blob/main/.github/workflows/build.yml#L162-L184. 

changes only needed in `release` job. And a single `SHA256SUM` file is required for extra release

Reply:
Does that mean putting the sha256sum of the two installation packages for Linux and drawin in the same file?

Like this:
```yaml
      - name: generate signature
        run: |
          sha256sum openmldb-*.tar.gz > SHA256SUM

      - name: Release
        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
        uses: softprops/action-gh-release@v1
        with:
          files: |
            openmldb-*.tar.gz
            SHA256SUM
```

Reply:
yes!